### PR TITLE
reset_password

### PR DESCRIPTION
**Pull request by weilingooi**
finish reset_password

### CODE REVIEW

**Comment by weilingooi** [comment]
ready

File: app/controllers/password_resets_controller.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  end
+
+  private
+
+  def update_password_params
+    params.require(:user).permit(:password, :password_confirmation).merge(reset_digest: nil)
+  end
+
+  def get_user
+    @user = User.find_by email: params[:email]
```
Comment (manhnt-0960) [inline app/controllers/password_resets_controller.rb:41]: please handle case @user = nil

File: app/controllers/password_resets_controller.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  end
+
+  def valid_user
+    return if @user && @user.authenticated? (:reset, params[:id])
```
Comment (manhnt-0960) [inline app/controllers/password_resets_controller.rb]:
```suggestion
    return if @user && @user.authenticated?(:reset, params[:id])
```

**Comment by vanvtt-0952** [comment]
commented

**Comment by weilingooi** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**